src/modules/json-ld/components/EventJsonLd.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import React from 'react'
+import { Helmet } from 'react-helmet'
+import { EventModel } from '@integreat-app/integreat-api-client'
+
+const createJsonLd = (event: EventModel) => {
+  const date = event.date
+
+  // https://developers.google.com/search/docs/data-types/event
+  const jsonLd: { endDate?: string, image?: string[], location: {} } = {
```

Comment:
I get that you added the type so you could add endDate and image later, but I think it is then kind of inconsistent to still add location as a property. Either extract the type completely (if you feel it is too much for a few lines) or remove location.

Reply:
You're right. I removed the `location` from the type, as I think, it's not really beneficial to define the type itself here... This would only make sense, if we could import the type from some authority defining it, but I haven't found one at first attempt...